resources/language/resource.language.en_gb/strings.po

```diff
@@ -1042,3 +1042,7 @@ msgstr ""
 msgctxt "#30987"
 msgid "No stream found, please try again!"
 msgstr ""
+
+msgctxt "#30988"
+msgid "InputStream Adaptive is not installed/enabled, pause/play and timeshift of live streams will not work"
```

Comment:
The phrasing confuses me because of the punctuation. I would change this into:

```suggestion
msgid "InputStream Adaptive is not available, therefore pause and timeshift of live streams will not work"
```

If ISA is not enabled, InputStream Helper will take care of it. Also, playing live streams will work so the statement was factual incorrect IMO.

Reply:
I agree for the most part but prefer to use "installed" instead of "available" because this gives the idea that it is missing and can't be installed.
```suggestion
msgid "InputStream Adaptive is not installed, therefore pause and timeshift of live streams may not work"
```